utils/doclint/generateDotnetApi.js

```diff
@@ -40,6 +39,11 @@ const nullableTypes = ['int', 'bool', 'decimal', 'float'];
 let documentation;
 /** @type {Map<string, string>} */
 let classNameMap;
+/** @type {Map<string, string>} */
+let knownNames = new Map([
```

Comment:
There are already a few "maps" around the code, can we call this `knownEnumValues` or something _meaningful_?

Reply:
how about now?